internal/controller/promotion/helm.go

```diff
@@ -125,7 +125,7 @@ func buildValuesFilesChanges(
 	tagsByImage := map[string]string{}
 	digestsByImage := make(map[string]string, len(images))
 	for _, image := range images {
-		tagsByImage[image.RepoURL] = image.Tag
+		tagsByImage[image.RepoURL] = "'" + image.Tag + "'"
```

Comment:
This will break things farther down on lines 155 to 161.

Just change line 160.